src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -83,7 +83,6 @@ class LaunchModes(Enum):
     MESHING_MODE = ("meshing", Meshing, True, [])
     PURE_MESHING_MODE = ("pure-meshing", PureMeshing, True, [])
     SOLVER = ("solver", Solver, False, [])
-    SOLVER_LITE = ("solver-lite", SolverLite, False, [])
```

Comment:
@hpohekar, please remove it from the documentation of launch_fluent() as well. Thanks.

Reply:
Done. Thank you.